winding order en tilematrixset (#48)
Geonovum/praktijkrichtlijn-vector-tiling: Documentatie.md

- Beschikbare Tilematrixset links
- De URI van de TileMatrixSet
- Het Coordinaat Referentie Systeem (een URI heeft de voorkeur).

> - Een dynamische link (URI) naar de inidividuele s.

Reviewer: Hier mist nog wat.

> - Een dynamische link (URI) naar de inidividuele s.

Author: ```suggestion
- Een dynamische link (URI) naar de inidividuele tegels.
```

### Aanbeveling: Overzooming
<div class="informative">

> Voor een tiles endpoint is in de OGC API een tileMatrixSet te definieren. Deze tilematrix kan voorzien worden van [tileMatrixsetlimits](https://docs.ogc.org/is/17-083r4/17-083r4.html#table12) met `maxTileRow` en `minTileRow` en `maxTileRow` en `minTileRow` `tileMatrix` level definitie zorgt ervoor dat de client kan overzoomen op de vector tiles. Dit houdt in dat, wanneer vector tiles niet beschikbaar zijn op een bepaald zoom level, een client de vector tiles op kan halen van een bovenliggend zoomniveau voor de tileMatrixSet. 

Author: yup, dat is een rabberige tekst,  ik heb het aangepast.

> Voor een tiles endpoint is in de OGC API een tileMatrixSet te definiëren. Deze tilematrix kan voorzien worden van [tileMatrixsetlimits](https://docs.ogc.org/is/17-083r4/17-083r4.html#table12). Hierin wordt doormiddel van `maxTileRow`, `minTileRow`, `maxTileCol` en `minTileCol` voor ieder `tileMatrix` aangegeven welke tegels beschikbaar zijn voor welke zoomlevels . Deze definitie zorgt ervoor dat de client kan overzoomen op de vector tiles. Dit houdt in dat, wanneer vector tiles niet beschikbaar zijn op een bepaald zoom level, een client de vector tiles op kan halen van een bovenliggend zoomniveau voor de tileMatrixSet. 

Reviewer: _iedere_ `tileMatrix`?

> Voor een tiles endpoint is in de OGC API een tileMatrixSet te definiëren. Iedere tilematrix kan voorzien worden van [tileMatrixsetlimits](https://docs.ogc.org/is/17-083r4/17-083r4.html#table12). Hierin wordt doormiddel van `maxTileRow`, `minTileRow`, `maxTileCol` en `minTileCol` voor ieder `tileMatrix` aangegeven welke tegels beschikbaar zijn voor welke zoomlevels . Deze definitie zorgt ervoor dat de client kan overzoomen op de vector tiles. Dit houdt in dat, wanneer vector tiles niet beschikbaar zijn op een bepaald zoom level, een client de vector tiles op kan halen van een bovenliggend zoomniveau voor de tileMatrixSet. 

Reviewer: ```suggestion
Voor een tiles endpoint is in de OGC API een tileMatrixSet te definiëren. Iedere tilematrix kan voorzien worden van [tileMatrixSetLimits](https://docs.ogc.org/is/17-083r4/17-083r4.html#table12). Hierin wordt door middel van `maxTileRow`, `minTileRow`, `maxTileCol` en `minTileCol` voor ieder `tileMatrix` aangegeven welke tegels beschikbaar zijn voor welke zoomlevels . Deze definitie zorgt ervoor dat de client kan overzoomen op de vector tiles. Dit houdt in dat, wanneer vector tiles niet beschikbaar zijn op een bepaald zoom level, een client de vector tiles op kan halen van een bovenliggend zoomniveau voor de tileMatrixSet. 
```

Verder raden wij aan ook te kijken naar de [[API-Designrules]] voor de Nederlandse overheid. Deze bevatten principes voor het aanbieden van APIs door Nederlandse overheidsorganisaties.

_AANBEVELING_ Volg de aanbevelingen over documentatie uit de [[API-Designrules]] voor de Nederlandse overheid: https://docs.geostandaarden.nl/api/API-Designrules/#documentation

> ### Aanbeveling: tile bestandsgrootte

Reviewer: Denk in deze paragraaf na over consistent gebruik van tegel versus tile. Gezien de rest van de tekst als zelfstandig naamwoord wel _tegel_ wordt gebruikt.

Reviewer 2: Voorstel voor regel 107/108: 
```
De bestandsgrootte van rastertiles wordt beprerkt door het aantal pixels (breedte en hoogte), de kleurdiepte en de variatie in pixelwaarden. De bestandsgrootte van vectortiles wordt bepaald door meerdere factoren:
- het aantal objecten dat in een vectortile zitten
- het aantal coördinaten dat de contouren van die objecten beschrijft
- het aantal atributen van het object
- de attribuutwaarden van het object

Hierom is het wenselijk om de bestandsgrootte van vectortiles in de gaten te beperken, bijvoorbeeld door objecten te selecteren, geometrisch te generaliseren, attributen te verwijderen of attribuutwaarden te vereenvoudigen.
```

Author: @emacgillavry @skinkie @jlschaap ik wilde dit gaan overnemen in de praktijkrichtlijn, maar realiseer me nu ook dat dit dan misschien beter past in de best practices zoals ik ze nu hier doorlopen heb: https://github.com/RoelvandenBerg/vector-tiling-best-practices/blob/roelvandenberg-aanvullingen/BestPractices.md#aanbeveling-optimalisatie-tegel-set

wat denken jullie?

Author: Als jullie dat vinden pas ik dat volgende week nog aan en merge ik jouw opmerkingen daar.

> In tegenstelling tot png tiles die een maximale grootte hebben kunnen vectortiles veel groter worden naar mate er meer vectoren in de tiles zitten. 

Reviewer: ```suggestion
In tegenstelling tot png tiles, die een maximale grootte hebben, kunnen vectortiles veel groter worden naar mate er meer vectoren in de tiles zitten. 
```

Verder raden wij aan ook te kijken naar de [[API-Designrules]] voor de Nederlandse overheid. Deze bevatten principes voor het aanbieden van APIs door Nederlandse overheidsorganisaties.

_AANBEVELING_ Volg de aanbevelingen over documentatie uit de [[API-Designrules]] voor de Nederlandse overheid: https://docs.geostandaarden.nl/api/API-Designrules/#documentation

### Aanbeveling: tile bestandsgrootte
De bestandsgrootte van rastertiles wordt beprerkt door het aantal pixels (breedte en hoogte), de kleurdiepte en de variatie in pixelwaarden. De bestandsgrootte van vectortiles wordt bepaald door meerdere factoren:
- het aantal objecten dat in een vectortile zitten
- het aantal coördinaten dat de contouren van die objecten beschrijft
- het aantal atributen van het object
- de attribuutwaarden van het object

> Het zorgvuldig kiezen van welke attributen worden opgenomen in tiles-datasets is van belang voor efficiënt gegevensbeheer. Door enkel de essentiële attributen toe te voegen, wordt onnodige gegevensuitwisseling voorkomen, wat de prestaties verbetert en de netwerkbelasting vermindert. Daarnaast kan het selectief opnemen van attributen waarop veelvuldig gefilterd wordt helpen bij het optimaliseren van de dataset voor specifieke gebruiksscenario's.

Reviewer: ```suggestion
Het zorgvuldig kiezen van welke attributen worden opgenomen in tiles-datasets is van belang voor efficiënt gegevensbeheer.
Door enkel de essentiële attributen toe te voegen, wordt onnodige gegevensuitwisseling voorkomen, wat de prestaties verbetert en de netwerkbelasting vermindert.
Daarnaast kan het selectief opnemen van attributen waarop veelvuldig gefilterd wordt helpen bij het optimaliseren van de dataset voor specifieke gebruiksscenario's.
```

> Het opvragen van extra informatie via een andere server op basis van een unieke identificatie die wel in de vectortegel beschikbaar is biedt real-time toegang tot actuele gegevens. Dit is vooral gunstig bij dynamische datasets, waarbij voorkomen wordt dat vectortiles herhaaldelijk gegenereerd moeten worden, of in situaties waar een hoge mate van actualiteit vereist is.

Reviewer: ```suggestion
Het opvragen van extra informatie via een andere server op basis van een unieke identificatie die wel in de vectortegel beschikbaar is biedt real-time toegang tot actuele gegevens.
Dit is vooral gunstig bij dynamische datasets, waarbij voorkomen wordt dat vectortiles herhaaldelijk gegenereerd moeten worden, of in situaties waar een hoge mate van actualiteit vereist is.
```
